packages/circuit-ui/components/Headline/Headline.tsx

```diff
@@ -25,7 +25,7 @@ type Size = 'one' | 'two' | 'three' | 'four';
 export interface HeadlineProps
   extends Omit<HTMLProps<HTMLHeadingElement>, 'size'> {
   /**
-   * A Circuit UI headline size.
+   * A Circuit UI headline size. Default `one`.
```

Comment:
Great 🙌 I think we should make this more of a pattern, always mention the default variant/size/etc

Reply:
Yeah, that's what I've been trying to do.